### PR TITLE
Adds missing changelog entries

### PR DESCRIPTION
**Pull request by maraisr**
We really should get in the habit of doing this in the PR's themselves, so we reference the PR author and number in the changelog correctly. So i've had to improvise in this scenario.

Thanks @acao

### CODE REVIEW

File: .changeset/config.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@1.5.0/schema.json",
-  "changelog": "@changesets/cli/changelog",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "graphql/graphiql" }
+  ],
```
Comment (maraisr) [inline .changeset/config.json:6]: This just formats the changelog entries a little nicer, puts in @<author> tags and PR numbers and other niceties. Most OSS projects that use Changesets use this.

cc @Andarist @mitchellhamilton

Comment (Andarist) [reply .changeset/config.json:6]: Yep - this is a cool generator that I'm recommending to be used by default. 👍

**Comment by acao** [comment]
yes we added the action but it didnt work retroactively

the issue with this PR is that there are package bumps. We just need the two md files. i can just re-create it after work!

**Comment by acao** [comment]
also, this was for #1776 and #1768. I can just redo this, my goal was just to create the two MD files sorry!

File: .changeset/config.json (hunk 1 above)
Comment (acao) [reply .changeset/config.json:6]: sounds great! can you open another PR to add this?

**Pull request closed without merging**